setup.py

```diff
@@ -36,6 +43,14 @@
 After paths to Arcade and Simulation discs are given, this script will unpack both discs and the VOL file from the Simulation Disc, then patch code and asset.
 The setup process may take some time, so please be patient and don't close this window even if the process seems stuck.
 """)
+if gui_mode:
+    print("About to open dialogs file paths (re-run script with -t to run without dialogs).")
+    if os.name == 'nt':
+        os.system("pause")
+    else:
+        os.system('read -s -n 1 -p "Press any key to continue ..."')
```

Comment:
Not a fan of this, any reason this cannot be just `input("About to open....")` with the return value discarded?

Reply:
```suggestion
    print("About to open dialogs file paths (re-run script with -t to run without dialogs).")
    input("Press Enter to continue ..."')
```
Not really, I was just making it so that you can't type characters there.